src/astro/dataframe/__init__.py

```diff
@@ -29,10 +30,18 @@ def dataframe(
     }
     if task_id:
         param_map["task_id"] = task_id
+    df_class = SqlDataframeOperator
+    if kwargs.get("_experimental", False):
+        from astro.sql.operators.sql_dataframe_refactor import (
+            SqlDataframeOperator as DFNew,
+        )
+
+        df_class = DFNew
+        kwargs.pop("_experimental")
```

Comment:
@tatiana @kaxil should we have a "experimental" flag per-task, or would it make more sense to just have an environment variable that automatically turns on experimental across the board? I can see it either way

Reply:
I would say once we have the SqlDecoratedOperator ready, we should just remove and replace the old table with the new one.

Reply:
+1, the `TruncateOperator` was fully replaced